### PR TITLE
Re-enable IIS tests

### PR DESCRIPTION
**Pull request by pakrym**
We've lost AppVerifier coverage by disabling these tests.

### CODE REVIEW

File: .azure/pipelines/ci.yml [hunk 1]
```diff
@@ -26,32 +27,23 @@ jobs:
...
+    buildArgs: "/p:SkipIISBackwardsCompatibilityTests=true /p:SkipIISTests=false /p:SkipIISExpressTests=true /p:SkipIISForwardsCompatibilityTests=true"
+- template: jobs/iisintegration-job.yml
+  parameters:
+    TestGroupName: IISExpress
+    buildArgs: "/p:SkipIISBackwardsCompatibilityTests=false /p:SkipIISTests=true /p:SkipIISExpressTests=false /p:SkipIISForwardsCompatibilityTests=true"
+- template: jobs/iisintegration-job.yml
+  parameters:
+    TestGroupName: IISForwardCompat
+    buildArgs: "/p:SkipIISBackwardsCompatibilityTests=true /p:SkipIISTests=true /p:SkipIISExpressTests=true /p:SkipIISForwardsCompatibilityTests=false"
+- template: jobs/iisintegration-job.yml
```
Comment (jkotalik) [inline .azure/pipelines/ci.yml]: Be consistent with how you define the templates.

**Comment by pakrym** [comment]
I'll wait for https://github.com/aspnet/AspNetCore/pull/5958

**Comment by natemcmaster** [comment]
#5958 is done.

File: build.ps1 [hunk 2]
```diff
@@ -228,6 +228,11 @@ if ($All) {
     $MSBuildArguments += '/p:BuildAllProjects=true'
 }
 elseif ($Projects) {
+    if (![System.IO.Path]::IsPathRooted($Projects))
+    {
+        $CurrentDirectory = Get-Location;
+        $Projects = Join-Path $CurrentDirectory $Projects
```
Comment (natemcmaster) [inline build.ps1]:
```suggestion
        $Projects = Join-Path (Get-Location) $Projects
```
you can avoid having a variable but just moving this inline. PowerShell is fun :)

File: src/Servers/IIS/build/repo.props [hunk 3]
```diff
@@ -16,24 +16,7 @@
   </PropertyGroup>
```
Comment (natemcmaster) [inline src/Servers/IIS/build/repo.props]: You can delete this entire file. I think this is left over from ProjectRef work. It's not used anymore. Likewise for src/Servers/IIS/build/repo.targets. I think @jkotalik is cleaning that up right now in the 2.2 branch.

**Review by natemcmaster — APPROVED** [review]

**Comment by natemcmaster** [comment]
👏 so close to getting tests green on PR checks. Nice work!

**Pull request merged**